helpdesk_mgmt_ticket_autoarchive/models/res_company.py

```diff
@@ -0,0 +1,9 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+    
+    autoarchive_tickets = fields.Integer(string='Days to archive ticket', default=30)
```

Comment:
Please add help attribute with a comment like this:
```suggestion
    autoarchive_tickets = fields.Integer(string='Days to archive ticket', default=30,
        help="If 0 is filled, this functionality will be disabled")
```

And add a constrain code in order to prevent negative values